pkg/config/controller/config-provider.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 const (
-	configMapName = "vm-import-controller-config"
+	ConfigMapName = "vm-import-controller-config"
```

Comment:
that requires a comment ? (due to init-cap)

Reply:
Fixed.